src/flow_solver/flow_solver.cpp

```diff
@@ -500,19 +505,23 @@ int FlowSolver<dim,nstate>::run() const
             }
 
             // update time step in flow_solver_case
-            flow_solver_case->set_time_step(time_step);
-
+            ode_solver->step_in_time(time_step,false);
+            
```

Comment:
If the step_in_time function has been moved ahead of the advance solution comment, can the comment be removed as it doesn't pertain to anything?